heat/scripts/run_ansible.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+OS="$(cat /etc/os-release |grep ^NAME=|tr [:upper:] [:lower:])"
```

Comment:
please change this comand with `cmd` and check the script with https://www.shellcheck.net/, there are other issues as well